template.yaml

```diff
@@ -149,7 +149,15 @@ Resources:
         openapi: 3.0.3
         info:
           title: NVA Public Search API
-          version: '0.1.0'
+          version: '0.2.0'
+          description: |-
+            Public available search API for resources in NVA. The API is backed by ElasicSearch and uses ElasticSearch query syntax.
```

Comment:
Public search API…Elast*T*icSearch and

Reply:
Maybe add link to ES-query syntax

Reply:
Fixed typo and added link to AWS supplied documentation